### PR TITLE
VIH-8207 Copy hearingId to Clipboard component - VHO CC

### PR DESCRIPTION
**Pull request by alberto-hortiguela**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/VIH-8207

### Change description ###
copy-id component fix to use hearing ID to copy to clipboard

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/copy-id/copy-id.component.ts [hunk 1]
```diff
@@ -1,24 +1,27 @@
...
 import { ClipboardService } from 'ngx-clipboard';
+import { VhoQueryService } from '../services/vho-query-service.service';
 
 @Component({
     selector: 'app-copy-id',
     templateUrl: './copy-id.component.html'
 })
 export class CopyIdComponent implements OnInit {
-    @Input() conference: HearingSummary;
+    @Input() conference: string;
```
Comment (robertbennett1998) [inline VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/copy-id/copy-id.component.ts:10]: Could you rename to conferenceId please

**Review by robertbennett1998 — CHANGES_REQUESTED** [review]
LGTM just a minor change :)

File: VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/copy-id/copy-id.component.ts (hunk 1 above)
Comment (alberto-hortiguela) [reply VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/copy-id/copy-id.component.ts]: That´s done now

**Review by krishnapottekkat-hmcts — APPROVED** [review]
LGTM.

File: VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/copy-id/copy-id.component.ts [hunk 2]
```diff
@@ -1,24 +1,27 @@
...
+    async ngOnInit() {
         this.resetText();
+        const response = await this.vhoQueryService.getConferenceByIdVHO(this.conferenceId);
+        this.hearingId = response.hearing_id;
     }
 
-    copyToClipboard(conference: HearingSummary) {
-        this.clipboardService.copyFromContent(conference.id);
+    copyToClipboard() {
+        this.clipboardService.copyFromContent(this.hearingId);
```
Comment (omio-ma) [inline VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/copy-id/copy-id.component.ts:24]: why are we changing this? This is different to the link we use for the quick link?

File: VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/vho-hearing-list/vho-hearing-list.component.html [hunk 3]
```diff
@@ -14,7 +14,7 @@
           class="copyTooltipText"
           id="copyTooltipText"
           >
-          <div><app-copy-id [conference]="conference"></app-copy-id></div>
+          <div><app-copy-id [conferenceId]="conference.id"></app-copy-id></div>
```
Comment (omio-ma) [inline VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/vho-hearing-list/vho-hearing-list.component.html:17]: any reason why we are changing this, as this is not related to quick links?
